assets/app/views/projects.html

```diff
@@ -1,61 +1,71 @@
-<div ng-if="!showGetStarted" class="container">
-  <div ng-if="(projects | hashSize) === 0" class="text-muted">Loading...</div>
-  <div ng-if="(projects | hashSize) !== 0">
-    <h1 style="display: inline-block;">Projects</h1>
-    <a ng-if="canCreate" href="createProject" style="margin-top: 10px;" class="btn btn-lg btn-primary pull-right">New Project</a>
-    <alerts alerts="alerts"></alerts>
-    <div ng-repeat="project in projects | orderByDisplayName">
-      <div row flex cross-axis="center" class="tile tile-project tile-click tile-flex">
-        <div flex>
-          <h2 class="project">
-            <a class="tile-target" href="project/{{project.metadata.name}}">{{(project | uniqueDisplayName : projects)}}</a>
-            <span ng-if="project.status.phase != 'Active'" data-toggle="tooltip" data-placement="right" title="This project has been marked for deletion." class="pficon pficon-warning-triangle-o" style="cursor: help; vertical-align: top; margin-left: 5px;"></span>
-          </h2>
-          <div class="muted" style="margin-top: -5px;" ng-if="project | description">
-            <truncate-long-text content="project | description" limit="512" use-word-boundary="true"></truncate-long-text>
+<default-header class="top-header"></default-header>
+<div class="wrap no-sidebar">
+  <div class="sidebar-left collapse navbar-collapse navbar-collapse-2">
+    <navbar-utility-mobile></navbar-utility-mobile>
+  </div>
+  <div class="middle surface-shaded">
+
+    <div ng-if="!showGetStarted" class="container">
+    <div ng-if="(projects | hashSize) === 0" class="text-muted" style="margin-top: 50px;">Loading...</div>
```

Comment:
I know you didn't add it, but we should probably remove the `text-muted` style since we don't use it on other loading messages.